### PR TITLE
feat: add read only tags for loopring

### LINKED ISSUE #2382: Introduce automatic read-only tags on the tag mechanism
## Abstract
With the introduction of L2 support with loopring we need to introduce automatic tags that are read-only and cannot be modified.
These tags can be only in the frontend and will be available to the account tag filtering.

## Motivation
Easily distinguish which accounts have which L2 protocol support.

- Make sure these tags appear properly also on the details page

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #2382

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Add read only tags for loopring

### CODE REVIEW

**Comment by lukicenturi** [comment]
`state.loopringBalances = {
  '0x106B62Fdd27B748CF2Da3BacAB91a2CaBaeE6dCa': {
    ETH: {
      amount: new BigNumber('10000'),
      usdValue: new BigNumber('10000')
    }
  }
};`

These code for testing loopring balances. Put it here `frontend/app/src/store/balances/mutations.ts:141`

File: frontend/app/src/types/user.ts [hunk 1]
```diff
@@ -181,11 +181,28 @@ export const Tag = z.object({
...
+  icon: z.string().default('').optional()
 });
 
 export type Tag = z.infer<typeof Tag>;
 
+export enum ReadOnlyTag {
+  LOOPRING = 'Loopring'
+}
+
+export const READ_ONLY_TAGS: { [key: string]: Tag } = {
```
Comment (kelsos) [inline frontend/app/src/types/user.ts:195]: any chance we would use [mapped types](https://www.typescriptlang.org/docs/handbook/2/mapped-types.html) to limit the possible key values?

**Comment by kelsos** [comment]
lgtm, the comment I have is minor so not blocking the PR

File: frontend/app/src/types/user.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/types/user.ts]: Updated @kelsos

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**